### DISCUSSION TITLE
IKEA DIRIGERA Hub will get Matter Controller support for IKEA's TRADFRI series (IKEA Home Smart platform)

### DISCUSSION THREAD

**Discussion by Hedda** [mfucci/node-matter#135] [General]
FYI, IKEA's new "DIRIGERA" hub with Matter Controller functionality is now available for purchase, it currently shipping firmware does however not yet run/expose Matter as not certified yet. (Only posted for discussion and reference as I don't personally use IKEA's gateway).

It has both an integrated Thread radio and an integrated Zigbee 3.0 radio (separate radio modules) so is meant to later be able to act as a Matter bridge for IKEA Trådfri series ("IKEA Home Smart" platform) products.

Their new hub it is meant to fully replace their previous/old IKEA TRÅDFRI Gateway (which only had a Zigbee radio and will not be upgraded to get Matter support) and is now listed as EOS (End-of-Sale).

https://www.ikea.com/us/en/p/dirigera-hub-for-smart-products-white-smart-50503414/

https://about.ikea.com/en/newsroom/2022/10/24/ikea-introduces-dirigera-and-the-new-ikea-home-smart-app

https://about.ikea.com/en/newsroom/2022/05/25/ikea-launches-dirigera-the-matter-ready-hub-for-smart-products-and-a-new-ikea-home-smart-app

Use case; it should be able to act as a Matter bridge with Matter Controller and Thread Border Router (but presubably only for IKEA's own branded smart systems products?).

https://www.ikea.com/us/en/cat/smart-systems-controls-53259/

https://www.ikea.com/us/en/cat/smart-home-hs001/

https://www.ikea.com/us/en/planners/smart-home/#/

https://www.ikea.com/us/en/customer-service/product-support/

IKEA currently sells a lot of Zigbee based products within the catagories of; smart lighting, smart blinds, smart air purifiers and accessories

PS: I believe IKEA was also one of the board members of the CSA (Connectivity Standards Alliance) and promoters of the Matter 1.0 standard:

https://csa-iot.org/members/

https://csa-iot.org/newsroom/ikea-joins-board/

**Comment by Apollon77** [comment]
Correct, I also have it already at home, but yes waiting for the FW upgrade ;-)
